src/unwinder/arm64/unwinder_arm64.cpp

```diff
@@ -215,7 +215,11 @@ Return Value:
         SourceAddress = StartingSp + FIELD_OFFSET(ARM64_KTRAP_FRAME, X);
         for (RegIndex = 0; RegIndex < 18; RegIndex++) {
             UPDATE_CONTEXT_POINTERS(UnwindParams, RegIndex, SourceAddress);
+#ifdef __clang__
```

Comment:
Thats probably correct, but I'm not sure on the entire approach I took here.  It might be better to unify the context decl with what ms has, but the anonymous structs pose a bit of an issue.